README.md

```diff
@@ -304,11 +304,13 @@ If you'd like to use a tool/language/framework that is not supported by our temp
 1) Your environment must include a Dockerfile that installs any requirements you may want.
 2) Custom models require a simple webserver in order to make predictions. We recommend putting this in
 your environment so that you can reuse it with multiple models. The webserver must be listening on port 8080 and implement the following routes:
-    1) `GET /{URL_PREFIX}/` This route is used to check if your model's server is running.
-    2) `POST /URL_PREFIX/predict/` This route is used to make predictions.
+   > **Note: `URL_PREFIX` is an environment variable that will be available at runtime. It has to be read and pasted into the routes.**
+    1) `GET /URL_PREFIX/` and `GET /URL_PREFIX/ping/` These routes are used to check if your model's server is running.
+    2) `GET /URL_PREFIX/health/` This route is used to check if model is loaded and functioning properly.
```

Comment:
/health route is not required for execution environment

IMO we should say that the required minimum is 
1. `/`
2. `/predict`

then, users can have "extensions":
- `/stats` to get memory usage measurements and charts in CM Testing Performance Check and Stability Check.
- `/health` to be able to properly propagate server initialisation errors to CM Testing Startup Check. Custom models are run in k8s, so if a container fails k8s will attempt to restart it. If model loading fails -- there's no point in failing the model container, because k8s will restart it, then model loading will fail again, and as a result container will go into crash/restart loop (for several minutes). To avoid such a case, if model loading loading fails - model container should not fail, instead the server should be up with `/` route, and the error should be returned via `/health` route with 513 response code.

Reply:
/health is ok
But for stats, we need schema?

Reply:
you can review again